start_docker_service.sh

```diff
@@ -1,4 +1,4 @@
-TAGNAME=4.0.0-4.6.0-salobj_5.0.0-aths_1.4.0
+TAGNAME=4.0.0-4.7.0-salobj_5.4.0-aths_1.4.0
 DOCKER_IMA=lsstdm/atheaderservice:$TAGNAME
 NAME=aths
```

Comment:
If you're renaming everything else from ATHeaderService to HeaderService, you might want to rename these too.

Reply:
Thanks for finding this one too.  The `lsstdm/atheaderservice` is a reserved space on the DM dockerhub that I'll need to have Josh change for me to ``lsstdm/headerservice`

Reply:
Got the new name from Josh `lsstdm/headerservice` for dockerhub